json/Side_Text.txt

```diff
@@ -9164,1039 +9164,1039 @@
 		"jp_text": "そうだっ！",
-		"tr_text": "",
+		"tr_text": "That's it!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "4",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "セラフィさんってば<%br>ランチャーが大好きだよな？",
-		"tr_text": "",
+		"tr_text": "Seraphy, you love launchers, don'cha?",
 		"fileID": 11
 	},
 	{
 		"eventNo": "5",
 		"jp_name": "セラフィ",
 		"tr_name": "Seraphy",
 		"jp_text": "は、はい。<%br>それが何か？",
-		"tr_text": "",
+		"tr_text": "Th-that's right.<%br>Why do you ask?",
 		"fileID": 11
 	},
 	{
 		"eventNo": "6",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "レッドスコルピオって、もともとの武器は<%br>ランチャーなんだろ？",
-		"tr_text": "",
+		"tr_text": "Red Scorpio's Origin Weapon is a launcher, right?",
 		"fileID": 11
 	},
 	{
 		"eventNo": "7",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "いかにランチャーがすごいかってことを<%br>レッドスコルピオに教えてほしいんだ。",
-		"tr_text": "",
+		"tr_text": "I wanna help her see just how great launchers are.",
 		"fileID": 11
 	},
 	{
 		"eventNo": "8",
 		"jp_name": "レッドスコルピオ",
 		"tr_name": "Red Scorpio",
 		"jp_text": "そ、そんな……<%br>わざわざ、申し訳ないです……",
-		"tr_text": "",
+		"tr_text": "Th-that...<%br>You don't have to do that for me...",
 		"fileID": 11
 	},
 	{
 		"eventNo": "9",
 		"jp_name": "セラフィ",
 		"tr_name": "Seraphy",
 		"jp_text": "わかりました！<%br>私に、任せてください！",
-		"tr_text": "",
+		"tr_text": "Understood!<%br>Leave it all to me!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "10",
 		"jp_name": "セラフィ",
 		"tr_name": "Seraphy",
 		"jp_text": "みなさんが任務に行っている間に<%br>しっかりと準備をしておきます！",
-		"tr_text": "",
+		"tr_text": "I'll get something together<%br>while you're on your mission!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "11",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "たのんだぜっ！<%br>それじゃみんな、任務にしゅっぱーつ！",
-		"tr_text": "",
+		"tr_text": "Awesome!<%br>OK everyone, let's get goin'!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "0",
 		"jp_name": "セラフィ",
 		"tr_name": "Seraphy",
 		"jp_text": "みなさん、任務完了お疲れ様でした。",
-		"tr_text": "",
+		"tr_text": "Team, good job on your mission.",
 		"fileID": 11
 	},
 	{
 		"eventNo": "1",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "おうっ！　それよりセラフィさん<%br>ランチャーの話、だいじょうぶか？",
-		"tr_text": "",
+		"tr_text": "Never mind that, didn't ya have<%br>somethin' to say about launchers?",
 		"fileID": 11
 	},
 	{
 		"eventNo": "2",
 		"jp_name": "セラフィ",
 		"tr_name": "Seraphy",
 		"jp_text": "まかせてください！<%br>話したいことをしっかりまとめておきました！",
-		"tr_text": "",
+		"tr_text": "Absolutely!<%br>I've put together a speech all about launchers!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "3",
 		"jp_name": "セラフィ",
 		"tr_name": "Seraphy",
 		"jp_text": "レッドスコルピオさん。<%br>私がランチャーに魅力を感じる理由ですが……",
-		"tr_text": "",
+		"tr_text": "So, Red Scorpio. The number one<%br>reason I find launchers so appealing...",
 		"fileID": 11
 	},
 	{
 		"eventNo": "4",
 		"jp_name": "セラフィ",
 		"tr_name": "Seraphy",
 		"jp_text": "それは、何といっても<%br>「一撃の威力」です！",
-		"tr_text": "",
+		"tr_text": "...is the enormous amount of<%br>power they put into every shot!",
```

Comment:
To make it punny, you could have done something like `...is that "explosion" of power they put into every shot!`